src/modules/Modal.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable camelcase */
+// eslint-disable-next-line max-classes-per-file
```

Comment:
- [ ] I recommend avoiding the use of `eslint disable` commands in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100: